docs/reference/chart-axis.md

```diff
@@ -1,3 +1,6 @@
+---
+meta_description: "Image-charts API propose to customize your graphics as for example with chart axis. This documentation shows you how to use the API url parameters to generate a chart."
```

Comment:
- "propose to" pas anglais
- "Learn how to customize your chart axis, from label styling to axis ranges."